tests/README.md

```
@@ -13,12 +13,33 @@ First activate the virtual environment:
 source ./venv/bin/activate
 ```
 
+Then start the development configuration of DFIR-IRIS server:
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)